src/goose/synopsis/moderator.py

```diff
@@ -33,19 +35,25 @@ class Synopsis(Moderator):
     goose state is managed.
     """
 
-    def __init__(self) -> None:
-        super().__init__()
+    def __init__(self, notifier: Optional[Notifier] = None) -> None:
+        super().__init__(notifier=notifier)
         self.current_summary = ""
         self.current_plan = ""
         self.originals = []
 
+        if self.notifier:
+            self.notifier.status("Loading hints...")
         hints = []
         hints_path = Path(".goosehints")
         home_hints_path = Path.home() / ".config/goose/.goosehints"
         if hints_path.is_file():
             hints.append(render_template(hints_path))
+            if self.notifier:
+                self.notifier.status("Loaded local hints")
         if home_hints_path.is_file():
             hints.append(render_template(home_hints_path))
+            if self.notifier:
+                self.notifier.status("Loaded global hints")
```

Comment:
might be extra stuff on the screen but if we show the full path to hints on iTerm (and maybe other terminal emulators) it'll create a hyperlink to click when a user holds down ⌘ cmd.

Reply:
@lamchau yeah this is only for moderating at the moment - but that is a cool idea- will add it elsewhere